src/app/containers/Figure/index.jsx

```diff
@@ -45,25 +53,39 @@ const FigureContainer = ({
     <Image alt={alt} src={src} width={width} srcset={srcset} />
   );
 
+  let Wrapper = NestedGridItemLarge;
+
+  if (height === width) {
+    Wrapper = NestedGridItemMedium;
+  }
+  if (height > width) {
+    Wrapper = NestedGridItemSmall;
+  }
+
   return (
-    <Figure>
-      <ImagePlaceholder ratio={ratio}>
-        {platform === 'amp' ? (
-          <AmpImg
-            alt={alt}
-            attribution={copyright || ''}
-            layout="responsive"
-            src={src}
-            height={height}
-            width={width}
-          />
-        ) : (
-          renderImage(imageToRender, lazyLoad)
-        )}
-        {renderCopyright(copyright)}
-      </ImagePlaceholder>
-      {renderCaption(captionBlock, type)}
-    </Figure>
+    <Wrapper
```

Comment:
Looks good so far.
You'll want to use this wrapper twice - around each of the ImagePlaceholder & Caption - with different group values depending on the designs.

Reply:
Makes sense, was wondering why the caption was not exacting fitting the designs mockups.

Reply:
Looking at the `styledGrid.jsx` file and the wrappers that are available:

In this outer wrapper, you'll want to use `NestedGridParentLarge`.
Then for the items that wrap the ImagePlaceholder and Caption, you'll want `NestedGridChildItemLarge`.

Reply:
@sareh made some updates, kindly review and check if am headed on the right direction

Reply:
@RayNjeri I've added comments below. Let me know if you have any further questions.